integration-test/src/main/java/org/apache/iotdb/itbase/env/BaseEnv.java

```diff
@@ -40,4 +45,14 @@ public interface BaseEnv {
   void setTestMethodName(String testCaseName);
 
   void dumpTestJVMSnapshot();
+
+  List<ConfigNodeWrapper> getConfigNodeWrapperList();
+
+  void setConfigNodeWrapperList(List<ConfigNodeWrapper> configNodeWrapperList);
+
+  List<DataNodeWrapper> getDataNodeWrapperList();
```

Comment:
Are methods `getDataNodeWrapperList` and `setDataNodeWrapperList` used?
If we will add more tests for DataNodes later, these methods can be retained.

Reply:
it will be used in DataNode test